pyproject.toml

```diff
@@ -89,6 +119,18 @@ line_length = 88
 multi_line_output = 3  # black interop
 use_parentheses = true
 
+[tool.tox]
+legacy_tox_ini = """
+[tox]
+isolated_build = true
+envlist = py36, py37, py38
+[testenv]
+whitelist_externals = poetry
+commands =
+    poetry install -v
+    poe test
+"""
+
```

Comment:
We're not using tox because our build matrix is built at CI, so please remove this part too.

Reply:
OK, updating the CONTRIBUTING.md to match as well then.